### PR TITLE
feat: disableaoidb

### PR DESCRIPTION
**Pull request by Faf4a**
read commits

disableAoiDB: true/false

### CODE REVIEW

**Comment by Leref** [comment]
update index.d.ts

**Comment by Faf4a** [comment]
done done

File: src/classes/AoiBase.js [hunk 1]
```diff
@@ -135,10 +132,6 @@ class BaseClient extends Discord.Client {
         }
     }
 
-    /**
```
Comment (Leref) [inline src/classes/AoiBase.js:138]: undo.

**Review by Leref — CHANGES_REQUESTED** [review]

File: src/classes/AoiBase.js (hunk 1 above)
Comment (Faf4a) [reply src/classes/AoiBase.js:138]: done

**Comment by Leref** [comment]
I just realized you didn't make it `=== true` otherwise it can be any type.

**Pull request merged**